e2e/Makefile

```diff
@@ -2,24 +2,25 @@ test-e2e: test-e2e-minimal test-e2e-individually test-e2e-complete
 
 build: 
 	npm run build
+	npm link
 
 test-e2e-minimal: build
 	rm -fr ./e2e/minimal && mkdir ./e2e/minimal && rsync -r ./examples/minimal/ ./e2e/minimal/
-	cd ./e2e/minimal && npm i && npx sls package
+	cd ./e2e/minimal && npm i && npm link serverless-esbuild && npx sls package
```

Comment:
huh, is this working then? I had unsuccessful attempts at using `npm link`

Reply:
aha yes I played with it a bit locally